tests/nested/manual/hybrid-remodel/task.yaml

```diff
@@ -16,7 +16,7 @@ environment:
 
 prepare: |
   VERSION="$(tests.nested show version)"
-  snap download --basename=pc-kernel --channel="$VERSION/beta" pc-kernel
+  snap download --basename=pc-kernel --channel="$VERSION/stable" pc-kernel
```

Comment:
Should this be squashed with the previous patch?

Reply:
i will clean up the commit history when it's approved, and thank you for review!